cdisc_rules_engine/services/data_services/dummy_data_service.py

```diff
@@ -121,7 +122,7 @@ def get_dataset_by_type(
         )
 
     def get_define_xml_contents(self, dataset_name: str) -> bytes:
-        pass
+        return self.define_xml
```

Comment:
Should this be converted to bytes before returning? I think by default it is a string

Reply:
Would the update I made be better since from_file_contents takes Union[bytes, str] ?

Reply:
I think it would be better just to convert the string to bytes to conform with what local data service does at the moment. This way there is less room for the test endpoint to have failures that aren't caught by the local data service